my-blueprints/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.js
```

Comment:
Please drop `my-blueprints` folder entirely. Unless you have a clear reasoning we need it.

Reply:
I removed it now